File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { InnoCyberAuthenticationSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await InnoCyberAuthenticationSDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- InnoCyberAuthentication SDK exists test

local sdk = require("inno-cyber-authentication_sdk")

describe("InnoCyberAuthenticationSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
